roles/common/tasks/container_status.yml

```diff
@@ -0,0 +1,13 @@
+---
+- name: "[TEST] verify container status"
+  ansible.builtin.shell:
+    cmd: |
+      podman ps -a --format "{{ '{{.Names}} {{.Status}}' }}" | grep "{{ container_name }}" | awk '{print $2}'
+  changed_when: false
```

Comment:
This needs a failed_when, if you're going to mark it with a TEST prefix, otherwise, it'll pass as long as the shell cmd returns something.

Alternatively, don't mark this task as a test, but mark the following task as a test.

Currently, we rely on the custom_logger to produce a file that gathers the test-ids=passed|failed output, so there should be a test ID in the second line of the task name.
Hopefully, we can remove this at some point soon, and add a check to report_result that checks the junit output instead.